redisinsight/api/src/modules/rdi/repository/local.rdi.repository.ts

```diff
@@ -1,12 +1,38 @@
 import { Injectable } from '@nestjs/common';
 import { InjectRepository } from '@nestjs/typeorm';
 import { Repository } from 'typeorm';
-import { classToClass } from 'src/utils';
+import { v4 as uuidv4 } from 'uuid';
+
 import { EncryptionService } from 'src/modules/encryption/encryption.service';
 import { ModelEncryptor } from 'src/modules/encryption/model.encryptor';
-import { RdiRepository } from 'src/modules/rdi/repository/rdi.repository';
 import { RdiEntity } from 'src/modules/rdi/entities/rdi.entity';
-import { Rdi } from 'src/modules/rdi/models';
+import { Rdi, RdiType } from 'src/modules/rdi/models';
+import { RdiRepository } from 'src/modules/rdi/repository/rdi.repository';
+import { classToClass } from 'src/utils';
+
+// mock data
+let mockRdiInstances: Rdi[] = [
```

Comment:
I guess better remove it. You can use it for development. What do use think ?

Reply:
I removed the mock here but left the mock request when clicking the `Rdi Instance` header button to exercise adding RDI Instances so that they also can be deleted.